v1/lib/server/metadataUtils.js

```diff
@@ -58,7 +58,7 @@ function extractMetadata(content) {
     } catch (err) {
       // Ignore the error as it means it's not a JSON value.
     }
-    metadata[key] = value;
+    metadata[key] = isNaN(value) ? value : value+'';
```

Comment:
If your aim is to convert into a string, you could just do `${value}` or `String(value)`